src/pages/index.md

```diff
@@ -1,4 +1,12 @@
 ---
+`keywords:
+  - Creative Cloud
+  - API Documentation
+  - ExtendScript
+  - SDKDocumentation
```

Comment:
@naydav  No space between SDK and Documentation ?

Reply:
Fixed